server/v2/commands.go

```diff
@@ -92,13 +91,16 @@ func Commands[AppT AppI[T], T transaction.Tx](
 			return nil
 		},
 	}
+	startCmd.SetContext(rootCmd.Context())
```

Comment:
just asking why we need that, thought sub command will use root command's context?

Reply:
Unfortunately that's not the case.

Reply:
See https://github.com/spf13/cobra/issues/1109